test/unit/decode_fixtures.test.cpp

```diff
@@ -186,9 +186,169 @@ TEST_CASE("Read feature multipolygon")
     CHECK(mpoly[0][0][4].y == 0);
 }
 
-TEST_CASE("Read invalid: missing geometry type field")
+TEST_CASE("Read: UNKNOWN geometry type, results in no features")
+{
+    std::string buffer = open_tile("test/mvt-fixtures/fixtures/016/tile.mvt");
+    auto fm = mapbox::vector_tile::decode_tile<std::int64_t>(buffer);
+    CHECK(fm.empty());
+}
+
+TEST_CASE("Read: missing geometry type field, results in feature collection and no layer")
 {
     std::string buffer = open_tile("test/mvt-fixtures/fixtures/003/tile.mvt");
     auto fm = mapbox::vector_tile::decode_tile<std::int64_t>(buffer);
     CHECK(fm.empty());
 }
+
+TEST_CASE("Read: missing layer extent is assigned the default value")
+{
+    std::string buffer = open_tile("test/mvt-fixtures/fixtures/009/tile.mvt");
+    auto fm = mapbox::vector_tile::decode_tile<std::int64_t>(buffer);
+    REQUIRE(fm.size() == 1);
+    REQUIRE(fm.end() != fm.find("hello"));
+    auto fc = fm["hello"];
+    REQUIRE(fc.size() == 1);
+    auto f = fc[0];
+    REQUIRE(f.geometry.is<mapbox::geometry::point<std::int64_t>>());
+    auto pt = f.geometry.get<mapbox::geometry::point<std::int64_t>>();
+    // just creating a point is a check that we are handling with a default extent,
+    // but we'll want to assert on extent somewhere in the future
```

Comment:
note to @flippmoke that we'll want the ability to ensure the default extent is what we expect

Reply:
This is not covered yet in this library and is a failing of the library -- not of our current set of code. This is a good test to add and we should keep in mind that we need to change this at some point going forward.